api/resource.rb

```diff
@@ -304,12 +306,12 @@ def exported_properties
     # Returns all of the properties that are a part of the self_link or
     # collection URLs
     def uri_properties
-      [@base_url, @__product.base_url].map do |url|
+      [@base_url, @__product.default_version.base_url].compact.map do |url|
```

Comment:
Please don't use the default version instead of the version the user specified

Reply:
Whoops, this was the result of a bad rebase! Thanks for catching